packages/esm-appointments-app/translations/en.json

```diff
@@ -168,6 +167,5 @@
   "visitType": "Visit Type",
   "vitals": "Vitals",
   "week": "Week",
-  "weekly": "Weekly",
   "yes": "Yes"
 }
```

Comment:
These changes seem unrelated?  Maybe you need to update your branch?

Reply:
This is autogenerated from the git push hooks. My guess is that some previous commit in the main branch neglected to include this change.